parodos-model-api/src/main/java/com/redhat/parodos/workflow/annotation/WorkFlowProperties.java

```diff
@@ -0,0 +1,17 @@
+package com.redhat.parodos.workflow.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+
+@Target({ METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface WorkFlowProperties {
+
+	String version() default "";
```

Comment:
Why not ${git.commit.id} by default?

Reply:
Because this is something that the user want to use on workflow-examples jar for example, and if we do here, the git  commit will be the service instead of the user jar commit!

Reply:
Ah! Ok I see, makes sense